packages/fuel-gauge/src/advanced-logging.test.ts

```diff
@@ -182,22 +214,45 @@ describe('Advanced Logging', () => {
     });
 
     it('when using ScriptTransactionRequest', async () => {
-      const callTest = await setupCallTest({ cache: false });
-      const configurable = await setupConfigurable({ cache: false });
-      const coverage = await setupCoverage({ cache: false });
+      using launched = await launchTestNode({
+        contractsConfigs: [
+          { deployer: AdvancedLoggingAbi__factory, bytecode: AdvancedLoggingAbiHex },
+          {
+            deployer: AdvancedLoggingOtherContractAbi__factory,
+            bytecode: AdvancedLoggingOtherContractAbiHex,
+          },
+          { deployer: CallTestContractAbi__factory, bytecode: CallTestContractAbiHex },
+          { deployer: ConfigurableContractAbi__factory, bytecode: ConfigurableContractAbiHex },
+          { deployer: CoverageContractAbi__factory, bytecode: CoverageContractAbiHex },
+        ],
+        walletsConfig: {
+          amountPerCoin: 500_000,
+        },
```

Comment:
```suggestion
```

[Default](https://github.com/FuelLabs/fuels-ts/blob/7ae80b8a4af7a2b770bb687edf1d9db671cc4a60/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L33) `amountPerCoin` is `10_000_000_000` so you don't need to define this.

Reply:
755f8e0